Kubernetes/IdentityService/test-isapi-kubernetes-manifest.yml

```diff
@@ -64,17 +64,17 @@ spec:
         - name: CLIENTS__0__REQUIRECLIENTSECRET
           value: "true"
         - name: CLIENTS__0__REDIRECTURLS__0
-          value: https://buyingcataloguedev.uksouth.cloudapp.azure.com/oauth/callback
+          value: https://buyingcataloguetest.uksouth.cloudapp.azure.com/oauth/callback
```

Comment:
it'd be public, wouldn't it?
https://buyingcataloguetestpublic.uksouth.cloudapp.azure.com